editors/emacs/files/site-start-app.el

```diff
@@ -11,6 +11,6 @@
 ;; Info-directory-list contains ${prefix}/share/info. See #32148.
 (setq Info-default-directory-list (cons "__PREFIX__/share/info" Info-default-directory-list))
 
-;; Look in MacPorts ${prefix} for tree-sitter parser libraries
-(when (boundp 'treesit-extra-load-path)
-  (setq treesit-extra-load-path (cons "__PREFIX__/lib" treesit-extra-load-path)))
+;; Use the OS X Emoji font for Emoticons
```

Comment:
In case anybody wonders, I've mistakenly remove this in 2416784c00f9a292768e12cc2233cb252290aa15. This is still very much needed for Emacs 29.